### ISSUE TITLE
Beamed rests

### ISSUE THREAD

**Issue by rettinghaus** [rism-ch/verovio#1440]
When beams end on rests, they don't have the correct ending points (because the beam treats it as down stemmed note). When starting on rests it's better, although here is a minor offset for the (missing) stem. 
(@lpugin as you are currently working on beams, I leave this to you.)

![beamedrests_001](https://user-images.githubusercontent.com/7693447/80886502-23c4db00-8cbe-11ea-874f-3270f6461d10.png)

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title>Beamed rests</title>
                <respStmt><persName>Klaus Rettinghaus</persName></respStmt>
            </titleStmt>
            <pubStmt>
              <date isodate="2020-05-02" />
            </pubStmt>
        </fileDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef>
                        <staffGrp>
                            <staffDef n="1" lines="5" clef.shape="G" clef.line="2" />
                        </staffGrp>
                    </scoreDef>
                    <section>
                        <measure n="1">
                            <staff n="1">
                                <layer n="1">
                                  <beam place="above">
                                    <note dur="8" oct="5" pname="c" />
                                    <rest dur="8" />
                                  </beam>
                                  <beam place="below">
                                    <note dur="8" oct="5" pname="c" />
                                    <rest dur="8" />
                                  </beam>
                                  <beam place="above">
                                    <rest dur="8" />
                                    <note dur="8" oct="5" pname="c" />
                                  </beam>
                                  <beam place="below">
                                    <rest dur="8" />
                                    <note dur="8" oct="5" pname="c" />
                                  </beam>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by rettinghaus** [comment]
LilyPond and Dorico align starting and ending points from beams on rests to the middle of the glyph. This is consistent and looks ok. Probably Verovio should follow this rule.

**Comment by rettinghaus** [comment]
fixed with #1711 1711

**Comment by craigsapp** [comment]
It would be useful to provide rendering examples of the fix when closing the issue so that the before and after images can be seen on the same page:   In this case since the before example is given in the issue, the after example should also be in the issue for efficiency.   The pull request #1711 includes the after example, which is great:

![image](https://user-images.githubusercontent.com/1819669/94654938-7dc9db00-0306-11eb-90c3-2e50c187d75a.png)

Although the verb tense is not implying that :-)

You can copy and paste the text that creates an image that is already posted, such as:

```
![image](https://user-images.githubusercontent.com/1819669/94654938-7dc9db00-0306-11eb-90c3-2e50c187d75a.png)
```

for this image.
